packages/mobile/src/harmony-native/foundations/shadows/shadows.ts

```diff
@@ -56,6 +56,8 @@ export const shadows = {
   flat: {
     elevation: 0,
     shadowRadius: 0,
+    shadowOpacity: 0,
```

Comment:
this seems unrelated to other fixes in this PR?

Reply:
yeah it seems very random, but that is so the all the shadow types have the same props, otherwise i would have had to do a lot of annoying checks in the component